Documentation/BuildInstructionsLadybird.md

```diff
@@ -175,6 +175,13 @@ open -W --stdout $(tty) --stderr $(tty) ./Build/ladybird/bin/Ladybird.app
 open -W --stdout $(tty) --stderr $(tty) ./Build/ladybird/bin/Ladybird.app --args https://ladybird.dev
 ```
 
+### Experimental GN build
+
+There is an experimental GN build for Ladybird. It is not officially supported, but it is kept up to date on a best-effort
+basis by interested contributors. See the [GN build instructions](../Meta/gn/README.md) for more information.
+
+In general, the GN build organizes ninja rules in a more compact way than the CMake build, and it may be faster on some systems.
```

Comment:
I'd personally mention that it creates only a single build dir for host and target, which is IMHO one big nice feature.